client.go

```diff
@@ -206,6 +206,15 @@ type Client interface {
 	// Get SB_Global table options
 	SBGlobalGetOptions() (map[string]string, error)
 
+	// Creates a new port group in the Port_Group table named "group" with optional "ports" added to the group.
+	PortGroupAdd(group string, ports []string, external_ids map[string]string) (*OvnCommand, error)
+	// Sets "ports" and/or "external_ids" on the port group named "group". It is an error if group does not exist.
+	PortGroupSet(group string, ports []string, external_ids map[string]string) (*OvnCommand, error)
```

Comment:
For most APIs of this library, the naming convention is "Update" for the whole row, "Set" for a specific column. Since this API updates multiple columns, it may be better named as PortGroupUpdate(...).

In addition, there is a column ACL. Would you want to include in this API so that it can be updated? Same for PortGroupAdd() API.

Reply:
I'll update the name. I was intending to just add the API support for the existing port-group functionality in this PR, and then add support for the ACL column when I added the generic ACL support that we discussed so I could test them together.  Is this okay with you?